test/functional/tools/validation_metadata_in_range.xml

```diff
@@ -0,0 +1,30 @@
+<tool id="validation_metadata_in_range" name="validation_metadata_in_range" profile="19.05" version="0.1">
+  <command>
+    echo "Hello World" > out1;
+  </command>
+  <inputs>
+    <param name="input1" type="data" format="fasta" label="fasta file with more than 10 sequences">
```

Comment:
should be "more than one"?

Reply:
https://github.com/fmareuil/galaxy/pull/2